server/graphql/v1/mutations/backyourstack.js

```diff
@@ -25,23 +25,7 @@ export async function dispatchOrder(orderId) {
       `Your BackYourStack order is already complete for this month. The next dispatch of funds will be on ${nextDispatchDate}`,
     );
   }
-
-  let dispatchedOrders;
-  try {
-    dispatchedOrders = await dispatchFunds(order);
-  } catch (err) {
-    console.error(err);
-    throw new Error(`Unable to dispatch funds to collectves.`);
-  }
-
-  if (dispatchedOrders) {
-    const currentDispatchDate = (subscription.data && subscription.data.nextDispatchDate) || new Date();
-    subscription.data = {
-      nextDispatchDate: getNextDispatchingDate(subscription.interval, currentDispatchDate),
-    };
-
-    await subscription.save();
-  }
-
-  return dispatchedOrders;
+  // Funds are dispatched asynchronously
+  backgroundDispatch(order, subscription);
+  return { dispatching: true };
```

Comment:
Looks a bit like an useless object in the end. Why not. I don't have something strongly better for now.